.gitignore

```diff
@@ -79,3 +79,6 @@ __debug_bin
 ./traffic_ops_golang
 # Created by TP/TO docker integration tests
 junit/
+go.mod
+go.sum
+vendor/modules.txt
```

Comment:
We should not do this. It makes the project no longer a Go module.

Reply:
Fixed